repo/incubating/elastic/README.md

```diff
@@ -0,0 +1,206 @@
+# Elastic
+
+Elasticsearch is a distributed, RESTful search and analytics engine. It is based on Apache Lucene.
+
+This Framework is deploying an Elasticsearch Cluster.
+
+## Prerequisites
+
+You need a Kubernetes cluster up and running and Persistent Storage available with a default `Storage Class` defined.
```

Comment:
When testing on Minikube I got a bunch of OOMs that I was able to fix by adding more memory. Maybe worth mentioning a minimum for resources that is required.

My Minikube was started e.g. with: `minikube start --vm-driver=hyperkit --cpus=6 --memory=12288 --disk-size=40g`

Reply:
hi @fabianbaier Meister , made the resource requests configurable. Can we do a quick test with minikube next week. Trying it with cpu=2, memory=6g, and disk-size=10g.

Reply:
I think we would need at least 9gb for it to work on Minikube. So testing it with

```
🔥  Creating hyperkit VM (CPUs=2, Memory=8192MB, Disk=10000MB) ...
```
seems to open a bottle neck not on Mem but on CPU.

```
myes-data-0     0/1     Init:0/2          0          2s
myes-data-0     0/1     Init:1/2          0          5s
myes-data-0     0/1     PodInitializing   0          9s
myes-data-0     1/1     Running           0          13s
myes-data-1     0/1     Pending           0          1s
myes-data-1     0/1     Pending           0          1s
```

and then looking whats hanging: 
```
  Warning  FailedScheduling  1s (x11 over 4m45s)  default-scheduler  0/1 nodes are available: 1 Insufficient cpu.
```

I've tested it with the following resources just fine:

```
🔥  Creating hyperkit VM (CPUs=6, Memory=12288MB, Disk=40000MB) ...
```

Reply:
@fabianbaier updated the readme with the info for minikube